foursight_core/react/api/react_routes.py

```diff
@@ -118,120 +119,141 @@ def reactapi_route_user_get_or_patch_or_delete(env: str, uuid: str) -> Response:
         """
         Returns the contents of the raw check_setup.json file.
         """
         return app.core.reactapi_checks_raw(app.current_request.to_dict(), env)
 
     @route("/{env}/checks-registry", authorize=True)
-    def reactapi_route_checks_registry(env: str) -> Response:
+    def reactapi_route_checks_registry(env: str) -> Response:  # noqa
         """
         Returns detailed registered checks functions.
         """
         return app.core.reactapi_checks_registry(app.current_request.to_dict(), env)
 
     @route("/{env}/lambdas", authorize=True)
-    def reactapi_route_lambdas(env: str) -> Response:
+    def reactapi_route_lambdas(env: str) -> Response:  # noqa
         """
         Returns detailed info on defined lambdas.
         """
         return app.core.reactapi_lambdas(app.current_request.to_dict(), env)
 
     @route("/{env}/gac/{env_compare}", authorize=True)
-    def reactapi_route_gac_compare(env: str, env_compare: str) -> Response:
+    def reactapi_route_gac_compare(env: str, env_compare: str) -> Response:  # noqa
         """
         Compares and returns diffs for values in the given two GACs.
         """
         return app.core.reactapi_gac_compare(app.current_request.to_dict(), env, env_compare=env_compare)
 
     @route("/{env}/aws/s3/buckets", authorize=True)
-    def reactapi_route_aws_s3_buckets(env: str) -> Response:
+    def reactapi_route_aws_s3_buckets(env: str) -> Response:  # noqa
         """
         Return the list of all AWS S3 bucket names for the current AWS environment.
         """
         return app.core.reactapi_aws_s3_buckets(app.current_request.to_dict(), env)
 
     @route("/{env}/aws/s3/buckets/{bucket}", authorize=True)
-    def reactapi_route_aws_s3_buckets_keys(env: str, bucket: str) -> Response:
+    def reactapi_route_aws_s3_buckets_keys(env: str, bucket: str) -> Response:  # noqa
         """
         Return the list of AWS S3 bucket key names in the given bucket for the current AWS environment.
         """
         return app.core.reactapi_aws_s3_buckets_keys(app.current_request.to_dict(), env, bucket=bucket)
 
     @route("/{env}/aws/s3/buckets/{bucket}/{key}", authorize=True)
-    def reactapi_route_aws_s3_buckets_key_contents(env: str, bucket: str, key: str) -> Response:
+    def reactapi_route_aws_s3_buckets_key_contents(env: str, bucket: str, key: str) -> Response:  # noqa
         """
         Return the content of the given AWS S3 bucket key in the given bucket for the current AWS environment.
         """
         return app.core.reactapi_aws_s3_buckets_key_contents(app.current_request.to_dict(), env, bucket=bucket, key=key)
 
+    @route("/{env}/accounts", authorize=True)
+    def reactapi_route_accounts(env: str) -> Response:  # noqa
+        """
+        Returns info on known accounts/environments as defined in an accounts.json file if present.
+        """
+        return app.core.reactapi_accounts(app.current_request.to_dict(), env)
```

Comment:
What kind of info is in there? Returning account information always makes me nervous because even if it doesn't contain sensitive info now, someone could add some not realizing they were publishing. I _assume_ this is just generic account metadata?

Reply:
Yes this is generic account metadata, not exposing anything that wasn't already exposed, and this route is also only accessible if authenticated and authorized.

Reply:
Yes. Added comments, with this list:
Here are the pertinent data points returns by this endpoint, none of them secret:
```
- Various version info (Python packages, Elasticsearch)
- Foursight URL
- Foursight deployed timestamp
- Foursight stage name
- Foursight current/default environment name (not all known environments but a count of them)
- Global application configuration name i.e. identity (also in Portal health URL output)
- S3 global environment bucket name
- S3 Bucket organization name
- AWS account ID and alias
- S3 encryption ID (also in Portal health URL output)
- Auth0 Client ID
- reCAPTCHA ID
- Elasticsearch hostname
- RDS database hostname
- Portal URL
- Portal deployed timestamp
- Portal health URL output (verbatim)
```